config/manager/manager.yaml

```diff
@@ -1,29 +1,31 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: controller-manager
+  name: operator
   namespace: system
   labels:
-    control-plane: controller-manager
+    control-plane: operator
```

Comment:
I guess this is overwritten by the labels from `kustomization.yaml`? If we want these changes, let's move then to that place.

Reply:
True, I moved the top-level resource labels to `kustomization.yaml`